<resources>
    <string name="app_name">ShelfyBooks</string>
    <string name="user_not_found">User not found</string>

    <!-- data module/Firebase Client Id -->
    <string name="default_web_client_id">586730003621-c5kvgnu0od8l7pa75lkgcvj86s0g7ioa.apps.googleusercontent.com</string>

    <!-- data module/Firebase Authentication Errors -->
    <string name="error_invalid_credentials">The credentials provided are invalid.</string>
    <string name="error_user_collision">This email is already associated with another account.</string>
    <string name="error_invalid_user">The user is no longer valid. Please sign in again.</string>
    <string name="error_unexpected_firebase">An unexpected Firebase authentication error occurred.</string>

    <!-- data module/Credential Manager Errors -->
    <string name="error_no_credentials">No credentials available. Please sign in.</string>
    <string name="error_user_canceled">Sign-in was canceled. Please try again.</string>
    <string name="error_interrupted">Sign-in was interrupted. Please try again.</string>
    <string name="error_unknown">An unknown error occurred. Please try again.</string>
    <string name="error_unexpected_credential">Unexpected error: %1$s</string>

    <!-- data module/Network Errors -->
    <string name="error_no_internet">No internet connection. Please check your network.</string>
    <string name="error_service_unavailable">Service is currently unavailable. Please try again later.</string>
</resources>